src/EnergyPlus/MixedAir.cc

```diff
@@ -4814,6 +4906,86 @@ namespace MixedAir {
 		return Residuum;
 	}
 
+	Real64
+	MultiCompControlTempResidual(
+		Real64 const OASignal, // Relative outside air flow rate (0 to 1)
+		Array1< Real64 > const & Par // par(1) = mixed node number
+		)
+	{
+
+		// FUNCTION INFORMATION:
+		//       AUTHOR         R. Raustad
+		//       DATE WRITTEN   Nov, 2016
+		//       MODIFIED
+		//       RE-ENGINEERED
+
+		// PURPOSE OF THIS FUNCTION:
+		// Calculates residual function TMixSetPoint - TMix.
+		// Economizer damper position (OASignal) is being varied to zero the residual.
+
+		// METHODOLOGY EMPLOYED:
+		// Simulate the OA System to determine actual mixed air condition, calculates the
+		// mixed air temperature given the outside air damper position.
+
+		// REFERENCES:
+
+		// Using/Aliasing
+		using Psychrometrics::PsyTdbFnHW;
+
+		// Return value
+		Real64 Residuum; // residual to be minimized to zero
+
+		// Argument array dimensioning
+
+		// Locals
+		// SUBROUTINE ARGUMENT DEFINITIONS:
+		// pao(1) = mixed air node number
+		// par(2) = relief air node number
+		// par(3) = outside air node number
+		// par(4) = mixed air flow rate
+		// par(5) = FirstHVACIteration
+		// par(6) = AirLoopNum index
+
+		// FUNCTION PARAMETER DEFINITIONS:
+		// na
+
+		// INTERFACE BLOCK SPECIFICATIONS
+		// na
+
+		// DERIVED TYPE DEFINITIONS
+		// na
+
+		// FUNCTION LOCAL VARIABLE DECLARATIONS:
+		int MixNode; // mixed air node number
```

Comment:
And it's also now more preferable to just declare local variables where they are used, and initialize them as you declare them.  Not a big deal, just noting for future.